interfaces/acados_matlab_octave/ocp_generate_casadi_ext_fun.m

```diff
@@ -223,6 +226,13 @@ function ocp_generate_casadi_ext_fun(model_struct, opts_struct)
     % build
     compile_command = sprintf('"%s" & %s', msvc_env, build_cmd);
 else % gcc
+    %Read the mex C compiler configuration and extract the location
+    cCompilerConfig = mex.getCompilerConfigurations('C');
```

Comment:
This command is not available in Octave.

However, it seems that Octave automatically adds a path to compatible gcc version to the PATH environment variable when Octave starts, so it should be possible just to skip all of this if octave is used